File: algo/src/test/java/com/hartwig/actin/algo/evaluation/infection/MeetsCovid19InfectionRequirementsTest.java
package com.hartwig.actin.algo.evaluation.infection;

import static com.hartwig.actin.algo.evaluation.EvaluationAssert.assertEvaluation;

import com.hartwig.actin.TestDataFactory;
import com.hartwig.actin.algo.datamodel.EvaluationResult;

import org.junit.Test;

public class MeetsCovid19InfectionRequirementsTest {

    @Test
    public void canEvaluate() {
        MeetsCovid19InfectionRequirements function = new MeetsCovid19InfectionRequirements();

        assertEvaluation(EvaluationResult.UNDETERMINED, function.evaluate(TestDataFactory.createMinimalTestPatientRecord()));
    }
}

File: algo/src/test/kotlin/com/hartwig/actin/algo/evaluation/infection/MeetsCovid19InfectionRequirementsTest.kt
package com.hartwig.actin.algo.evaluation.infection

import com.hartwig.actin.algo.evaluation.EvaluationAssert.assertEvaluation
import com.hartwig.actin.datamodel.TestPatientFactory
import com.hartwig.actin.datamodel.algo.EvaluationResult
import org.junit.jupiter.api.Test

class MeetsCovid19InfectionRequirementsTest {

    private val function = MeetsCovid19InfectionRequirements()

    @Test
    fun `Should pass always`() {
        assertEvaluation(EvaluationResult.PASS, function.evaluate(TestPatientFactory.createMinimalTestWGSPatientRecord()))
    }
}